shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1388,6 +1426,45 @@ public static WireCommand readFrom(ByteBufInputStream in, int length) throws IOE
         }
     }
 
+    @Data
+    public static final class SegmentsMergedBatch implements Reply, WireCommand {
```

Comment:
`SegmentsBatchMerged` sounds better?

I'm saying this because the convention used in this class is:
- Verb + Target for Requests
- Target + Verb (i..e, "Merged", "Appended", "Sealed") for their respective responses.